### PR TITLE
"octal" plain error for strings(?)

### PR DESCRIPTION
**Pull request by PallHaraldsson**

### CODE REVIEW

**Review by yuyichao — CHANGES_REQUESTED** [review]
The old text is correct.

Octal escape sequence does not have a prefix other than the "escape" (i.e. `\`).

```julia
julia> "\123"
"S"
```

File: doc/src/manual/strings.md [hunk 1]
```diff
@@ -782,7 +782,7 @@ lets you use string notation to express literal byte arrays -- i.e. arrays of
 [`UInt8`](@ref) values. The rules for byte array literals are the following:
 
   * ASCII characters and ASCII escapes produce a single byte.
-  * `\x` and octal escape sequences produce the *byte* corresponding to the escape value.
+  * `\x` and hexadecimal escape sequence produce the *byte* corresponding to the escape value.
```
Comment (ararslan) [inline doc/src/manual/strings.md]: "sequences" was correct here, "sequence" is not

**Comment by yuyichao** [comment]
Note that my request for change instead of closing is because the octal escape sequence isn't actually being specified here.

**Comment by StefanKarpinski** [comment]
As you said, the old text is correct, so what's to do here?

**Comment by yuyichao** [comment]
Mentions that octal escape sequence means `\nnn`?

**Pull request closed without merging**